### PR TITLE
sanja

### PR DESCRIPTION
**Pull request by tomaszdrozdz**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/sanja/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6b87c2a2780abde59bdd6e13bcfe91f11d4217a192f1f401f83312a6b145420
+
+build:
+  #noarch: python
+  number: 0
+  skip: True  # [py<38]
```
Comment (scopatz) [inline recipes/sanja/meta.yaml:15]: This can be noarch python with `python >=3.8` below

Comment (tomaszdrozdz) [reply recipes/sanja/meta.yaml:15]: At first I had:

    build:
        noarch: python
        number: 0
        skip: True  # [py<38]
        script: "{{ PYTHON }} -m pip install . -vv"

    requirements:
        host:
            - python
            - pip
    run:
        - python >=3.8
        - sanic
        - jinja2

But I got 
> noarch packages can't have selectors. If the selectors are necessary, please remove noarch: python

File: recipes/sanja/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (marcelotrevisani) [inline recipes/sanja/meta.yaml:20]:
```suggestion
    - python >=3.8
```

File: recipes/sanja/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    - pip
+  run:
+    - python
```
Comment (marcelotrevisani) [inline recipes/sanja/meta.yaml:23]:
```suggestion
    - python >=3.8
```

File: recipes/sanja/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    - sanic
+    - jinja2
+
+test:
+  imports:
```
Comment (marcelotrevisani) [inline recipes/sanja/meta.yaml:28]:
```suggestion
  imports:
    - sanja
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/sanja/meta.yaml (hunk 1 above)
Comment (marcelotrevisani) [inline recipes/sanja/meta.yaml:15]: I believe Anthony meant the suggestions which I added
```suggestion
```

File: recipes/sanja/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: b6b87c2a2780abde59bdd6e13bcfe91f11d4217a192f1f401f83312a6b145420
+
+build:
+  #noarch: python
```
Comment (marcelotrevisani) [inline recipes/sanja/meta.yaml:13]:
```suggestion
```

File: recipes/sanja/meta.yaml (hunk 1 above)
Comment (scopatz) [reply recipes/sanja/meta.yaml:15]: The last one

**Comment by tomaszdrozdz** [comment]
Thank Yo for help and tips. :)

**Pull request closed without merging**